tracee/printer.go

```diff
@@ -167,6 +180,42 @@ func (p tableEventPrinter) Epilogue(stats statsStore) {
 	fmt.Fprintf(p.out, "Stats: %+v\n", stats)
 }
 
+
+type templateEventPrinter struct {
+	tracee        *Tracee
+	out           io.Writer
+	err           io.Writer
+	containerMode bool
+	templatePath  string
+}
+
+func (p templateEventPrinter) Init() {}
+
+func (p templateEventPrinter) Preamble() {}
+
+func (p templateEventPrinter) Error(err error) {
+	fmt.Fprintf(p.err, "%v", err)
+}
+
+
+func (p templateEventPrinter) Print(event Event) {
+	tmplPath := p.templatePath
+	if tmplPath != "" {
+		tmpl, err := template.ParseFiles(tmplPath)
```

Comment:
It would be better to parse the template in the `Init` rather then on every event

Reply:
Okay sure. I've made the change accordingly.